### PR TITLE
document how to disable chunking

### PR DESCRIPTION
**Pull request by jnweiger**
@TheOneRing reading the source, can you confirm, that this should disable chunking?

### CODE REVIEW

**Comment by jnweiger** [comment]
Then please leave a pointer: 'See [FAQ](URL) for more details'
The FAQ did not come up when googling for 'owncloud chunking environment variables'.

**Comment by mmattel** [comment]
My pov, this perfectly fits where it is made and a approval / merging would be totally fine

File: docs/modules/ROOT/pages/advanced_usage/environment_variables.adoc [hunk 1]
```diff
@@ -19,6 +19,7 @@ The environment variables are:
 | 10000000 (or 10 MB)
 | Specifies the initial chunk size of uploaded files in bytes.
 The client will dynamically adjust this size within the maximum and minimum bounds (see below).
+To disable chunking completely, set these four variables to 0: `OWNCLOUD_CHUNK_SIZE=0 OWNCLOUD_MAX_CHUNK_SIZE=0 OWNCLOUD_MIN_CHUNK_SIZE=0 OWNCLOUD_CHUNKING_NG=0`
```
Comment (ogoffart) [inline docs/modules/ROOT/pages/advanced_usage/environment_variables.adoc]: I think setting OWNCLOUD_CHUNK_SIZE=0 should be enough to disable chunking.
What makes you think you need to set all of this. Did you try it and OWNCLOUD_CHUNK_SIZE=0 was not enough?

Comment (jnweiger) [reply docs/modules/ROOT/pages/advanced_usage/environment_variables.adoc]: Not tried. I suggested all 4 of them, just to be sure. 
What should the user check, to learn if chunking is on or off?

Comment (ogoffart) [reply docs/modules/ROOT/pages/advanced_usage/environment_variables.adoc]: OWNCLOUD_CHUNK_SIZE=0 is the only required one that disable the chunking completely.

Setting `OWNCLOUD_MIN_CHUNK_SIZE`  or `OWNCLOUD_MAX_CHUNK_SIZE` to 0 would be a very bad idea as it would mean that the client will want to send an infinite amount of chunk of size 0. These are only used in the computation of the chunk size and not in the decision to use chunking or not.
Setting `OWNCLOUD_CHUNKING_NG` would just force a chunking algorithm but not decide wether or not use chunks.

So in other words, only `OWNCLOUD_CHUNK_SIZE=0` is required and should be documented.

**Comment by mmattel** [comment]
Ping, any progress ?

File: docs/modules/ROOT/pages/advanced_usage/environment_variables.adoc (hunk 1 above)
Comment (michaelstingl) [inline docs/modules/ROOT/pages/advanced_usage/environment_variables.adoc]:
```suggestion
To disable chunking completely, set `OWNCLOUD_CHUNK_SIZE=0` .
```

Comment (mmattel) [reply docs/modules/ROOT/pages/advanced_usage/environment_variables.adoc]: Suggestion has been incorporated

**Comment by mmattel** [comment]
@settermjd mind to take a look why the pdf build in drone is failing?

**Comment by mmattel** [comment]
Seems that another font has an issue:
`No such file or directory - mplus-1mn-regular.ttf not found in fonts`

File: docs/resources/themes/owncloud-theme.yml [hunk 2]
```diff
@@ -5,20 +5,20 @@ font:
       normal: notoserif-regular.ttf
       bold: notoserif-bold.ttf
       italic: notoserif-italic.ttf
-      bold_italic: notoserif-bold_italic.ttf
+      bold_italic: notoserif-bolditalic.ttf
```
Comment (TheOneRing) [inline docs/resources/themes/owncloud-theme.yml]: Why are the fonts touched here?

**Comment by TheOneRing** [comment]
I rebased the code

File: docs/resources/themes/owncloud-theme.yml (hunk 2 above)
Comment (settermjd) [reply docs/resources/themes/owncloud-theme.yml]: Because the configuration file uses incorrect font file names.

**Comment by settermjd** [comment]
> Seems that another font has an issue:
> `No such file or directory - mplus-1mn-regular.ttf not found in fonts`

I don't get the error locally. Not sure how the Drone build and local build work differently, yet.
